pulseapi/entries/views.py

```diff
@@ -149,7 +149,7 @@ class EntriesListView(ListCreateAPIView):
 
     #Query Parameters -
 
-    - `?search=` - Allows search terms
+    - `?search=` - Search by title, description, creator, or tag.
```

Comment:
Updated to use "and".

I don't think publishers are useful, at least at this point in time, especially since it's just staff that can publish at the moment.

Agreed, it may not be ideal, but it's fine for MVP.